services/marathon/marathon.go

```diff
@@ -206,10 +217,20 @@ func createApps(tasksById map[string][]marathonTask, marathonApps map[string]mar
 		tasks := []Task{}
 		for _, mTask := range tasksById[appId] {
 			if len(mTask.Ports) > 0 {
+				// If task is within grace period, it has no
+				// healthCheckResults but startedAt is nil, otherwise
+				// startedAt is a timestamp string
+				var healthy = (len(mTask.StartedAt) > 4)
```

Comment:
Just for reference, you can omit the wrapping parenthesis. Not a critique just a random fact for @drewrobb